places-solution-integrations/analytics/README.md

```diff
@@ -0,0 +1,6 @@
+# Places Integration with Adobe Analytics
+
+Below is the list of current types of integration between Adobe Places and Adobe Analytics:
```

Comment:
- Please reword as follows:
"Here is a list of the current integration types between Adobe Places and Adobe Analytics: